repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/intrafilterlogging/header-normalization.cfg.xml

```diff
@@ -22,14 +22,6 @@
             </whitelist>
         </target>
 
-        <!-- applies to all POST and PUT requests -->
-        <target http-methods="POST PUT">
-            <whitelist id="modification">
-                <header id="X-THIRD-Filter"/>
-                <header id="x-auth-token"/>
-            </whitelist>
-        </target>
-
```

Comment:
So personally i think functional tests weise there should be multiple tests, a set where it's in as much isolation as we can manage, and a set that more closely resembles it's live environment amongst a bunch of filters.